mlir/utils/jenkins/Jenkinsfile

```diff
@@ -851,7 +851,13 @@ pipeline {
                                 // Run MLIR perf benchmarks
                                 sh 'python3 ./bin/perfRunner.py'
                                 sh """python3 ./bin/perfRunner.py --op=gemm --batch_all \
-					--configs_file=${WORKSPACE}/mlir/utils/performance/bert-configs \
+                                       --configs_file=${WORKSPACE}/mlir/utils/performance/bert-configs \
+                                       --tuning_db=${WORKSPACE}/mlir_tuning_${CHIP}.tsv""" \
```

Comment:
This tuning db does not incoporate unet instances.
It needs to be added here as well : 

https://github.com/ROCmSoftwarePlatform/rocMLIR/blob/a997d5f51314b45d7a4c04f1599966dcf53f9b4d/mlir/utils/jenkins/Jenkinsfile#L603-L611

Need same for convs.

Reply:
The tsv's are appended

Reply:
I think I've done what you asked in the next commit.  Running a private Jenkins test now.